forge-gui/res/cardsfolder/g/golden_argosy.txt

```diff
@@ -4,7 +4,7 @@ Types:Legendary Artifact Vehicle
 PT:3/6
 T:Mode$ Attacks | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigFlicker | TriggerDescription$ Whenever CARDNAME attacks, exile each creature that crewed it this turn. Return them to the battlefield tapped under their owner's control at the beginning of the next end step.
 SVar:TrigFlicker:DB$ ChangeZoneAll | ChangeType$ Creature.CrewedThisTurn | Origin$ Battlefield | Destination$ Exile | RememberLKI$ True | SubAbility$ DelTrig
-SVar:DelTrig:DB$ DelayedTrigger | Mode$ Phase | Phase$ End of Turn | Execute$ TrigReturn | RememberObjects$ RememberedLKI | SubAbility$ DBCleanup | TriggerDescription$ Return them to the battlefield tapped under their owner's control at the beginning of the next end step.
+SVar:DelTrig:DB$ DelayedTrigger | Mode$ Phase | Phase$ End of Turn | Execute$ TrigReturn | RememberObjects$ RememberedLKI | TriggerDescription$ Return them to the battlefield tapped under their owner's control at the beginning of the next end step.
```

Comment:
I could be wrong, but I think you want to add a DBCleanup line with ClearRemembered